api/src/jvmMain/java/com/buenaflor/kflogger/parser/PrintfMessageParser.java

```diff
@@ -79,7 +79,7 @@ static String getSafeSystemNewline() {
    * @param formatStart the index of the (first) format character in the term.
    * @return the index after the last character of the term.
    */
-  abstract int parsePrintfTerm(
+  protected abstract int parsePrintfTerm(
```

Comment:
Small change to the Java Flogger codebase

Reply:
This change makes sense. In an ideal world it would make sense to make this internal so that this is only accessible to friend modules.